pkg/cli/image/mirror/mirror.go

```diff
@@ -745,11 +748,23 @@ func copyBlob(ctx context.Context, plan *workPlan, c *repositoryBlobCopy, blob d
 		}
 		plan.BytesCopied(n)
 		return nil
-	}()
-	if err != nil {
-		return err
 	}
-	return nil
+
+	return retry.OnError(
+		wait.Backoff{
+			Duration: time.Second,
+			Factor:   1,
+			Steps:    5,
+		},
+		func(err error) bool {
+			if strings.Contains(err.Error(), "REFUSED_STREAM") {
```

Comment:
`return strings.Contains(err.Error(), "REFUSED_STREAM")`? It seems simpler and you can always log in `copyFn`, especially that the error is expected one.

Reply:
Yeah, I am planning to remove this log - as soon as I manage to replicate the issue with `master`. I will get back to you.

Reply:
Will it be possible to understand that `oc` hit the error (with debug logging enabled)?

Reply:
No, I have removed the log. Do you want me to put it back?

Reply:
I changed my mind, there is value in having this logged. Following @soltysh idea I have chosen to log in the function body instead, PTAL.

https://github.com/openshift/oc/pull/706/files#diff-1e91260f74d6221c0e5d1b5fcf70982d26d168e294254fea0b6baf8b771f7bcaR741